lms/djangoapps/discussion/toggles.py

```diff
@@ -45,3 +45,16 @@
     'enable_moderation_reason_codes',
     __name__,
 )
+
+# .. toggle_name: discussions.enable_reported_content_email_notifications
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to toggle email notifications for reported content for moderators
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2022-03-08
+# .. toggle_target_removal_date: 2022-12-31
+REPORTED_CONTENT_EMAIL_NOTIFICATIONS_FLAG = CourseWaffleFlag(
```

Comment:
ENABLE_REPORTED_CONTENT_EMAIL_NOTIFICATIONS to make it consistent with the flag text